packages/edit-post/src/components/sidebar/settings-header/index.js

```diff
@@ -27,66 +30,51 @@ const SettingsHeader = ( { sidebarName } ) => {
 		};
 	}, [] );
 
-	const [ documentAriaLabel, documentActiveClass ] =
+	const documentAriaLabel =
 		sidebarName === 'edit-post/document'
 			? // translators: ARIA label for the Document sidebar tab, selected. %s: Document label.
-			  [ sprintf( __( '%s (selected)' ), documentLabel ), 'is-active' ]
-			: [ documentLabel, '' ];
+			  sprintf( __( '%s (selected)' ), documentLabel )
+			: documentLabel;
 
-	const [ blockAriaLabel, blockActiveClass ] =
+	const blockAriaLabel =
 		sidebarName === 'edit-post/block'
 			? // translators: ARIA label for the Block Settings Sidebar tab, selected.
-			  [ __( 'Block (selected)' ), 'is-active' ]
+			  __( 'Block (selected)' )
 			: // translators: ARIA label for the Block Settings Sidebar tab, not selected.
-			  [ __( 'Block' ), '' ];
+			  __( 'Block' );
 
-	const [ templateAriaLabel, templateActiveClass ] =
+	const templateAriaLabel =
 		sidebarName === 'edit-post/document'
-			? [ __( 'Template (selected)' ), 'is-active' ]
-			: [ __( 'Template' ), '' ];
+			? __( 'Template (selected)' )
+			: __( 'Template' );
 
-	/* Use a list so screen readers will announce how many tabs there are. */
 	return (
-		<ul>
-			{ ! isTemplateMode && (
-				<li>
-					<Button
-						onClick={ openDocumentSettings }
-						className={ `edit-post-sidebar__panel-tab ${ documentActiveClass }` }
-						aria-label={ documentAriaLabel }
-						data-label={ documentLabel }
-					>
-						{ documentLabel }
-					</Button>
-				</li>
-			) }
-			{ isTemplateMode && (
-				<li>
-					<Button
-						onClick={ openDocumentSettings }
-						className={ `edit-post-sidebar__panel-tab ${ templateActiveClass }` }
-						aria-label={ templateAriaLabel }
-						data-label={ __( 'Template' ) }
-					>
-						{ __( 'Template' ) }
-					</Button>
-				</li>
-			) }
-			<li>
-				<Button
+		<>
+			<Tabs.TabList>
+				<Tabs.Tab
+					id={ 'edit-post/document' }
+					onClick={ openDocumentSettings }
+					aria-label={
+						isTemplateMode ? templateAriaLabel : documentAriaLabel
+					}
+					data-label={
+						isTemplateMode ? __( 'Template' ) : documentLabel
+					}
```

Comment:
> I'll look a bit more to try and determine if/how it's in use, then if it's just tests, update them accordingly.

Sounds good. Alternatively, we could also just delete it and see if tests fail because of it.

> Do you specifically mean appending "(Selected)" to the `aria-label`? Or just that we'll have a valid `aria-selected` attribute applied to the current tab, so assistive tech is appropriately notified?

The latter. What I meant is that passing a custom `aria-label` attribute shouldn't be necessary anymore, since we're now using `Tabs` (which takes care of this internally by setting the `aria-selected="true"` attribute as needed).